app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -224,6 +225,7 @@ class WebViewActivity : BaseActivity(), io.homeassistant.companion.android.webvi
     private var downloadFileUrl = ""
     private var downloadFileContentDisposition = ""
     private var downloadFileMimetype = ""
+    private var javascriptInterface = "externalApp"
```

Comment:
this can be a `val` as its not going to be changed

Reply:
done